orttraining/orttraining/python/training/ortmodule/ortmodule.py

```diff
@@ -65,15 +65,32 @@ def forward(self, *inputs, **kwargs):
         '''Dummy documentation for forward method'''
         ...
 
+    def _apply(self, fn):
```

Comment:
how about `apply()`? or `eval()`? Did you searched for all `torch.nn.Module` APIs we need to override?

Reply:
```eval``` deletegates to ```train``` which has been overridden. I didn't see use for ```apply``` for the time being, but can add it here as well.
I'll go through all the methods once again and add them here.